v3/integration/lints/main.go

```diff
@@ -0,0 +1,77 @@
+package main // import "github.com/zmap/zlint/v3/integration/lints"
+
+/*
+ * ZLint Copyright 2021 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/zmap/zlint/v3/integration/lints/lint"
+
+	"github.com/zmap/zlint/v3/integration/lints/lints"
+)
+
+var Linters = []lint.Lint{
+	&lints.InitFirst{},
+}
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Printf("USAGE ./main <path to lint directory>\n")
+		os.Exit(1)
+	}
+	results, err := run(os.Args[1])
+	if err != nil {
+		fmt.Printf("A fatal error has occurred: %v\n", err)
+		os.Exit(2)
+	}
+	exitCode := 0
+	if len(results) != 0 {
+		exitCode = 1
+		fmt.Printf("Found %d linting errors\n", len(results))
+	}
+	for _, result := range results {
+		fmt.Printf("%s\n", result)
+	}
+	os.Exit(exitCode)
```

Comment:
I think it would be slightly cleaner if you reworked this to be something like:

```go
if len(results) == 0 {
  os.Exit(0)
}
fmt.Printf("Found %d linting errors\n", len(results))
for _, result := range results {
     fmt.Printf("%s\n", result)
}
os.Exit(1)
```

Reply:
I agree, dunno what I was thinking with the other one.